enterprise/frontend/src/embedding-sdk/package.template.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@metabase/embedding-sdk-react",
-  "version": "0.0.3",
-  "description": "Metabase Embedding SDK",
+  "version": "0.1.0",
+  "description": "Metabase React Embedding SDK",
```

Comment:
should we use the same wording in Readme as well? I used `Metabase Embedding SDK for React` there.

I think we either use this one or another.

Reply:
Good catch - updated this. Thanks!